app/models/user.rb

```diff
@@ -2,4 +2,14 @@ class User < ActiveRecord::Base
   include Clearance::User
 
   has_many :itineraries
+
+  def testing_hound
+
+    true ? 'hi' : "bye"
+
+    puts "slkajfksdljflkdsaj fkldsaj fkljasdlk fjsadlk jfalksdj flkasdjf lkajsdkl fjsadlk fjlaksj d"
+
+    return {:hi => :bye}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Redundant `return` detected.<br>Space inside { missing.<br>Space inside } missing.